index.js

```diff
@@ -2,18 +2,17 @@
 
 const arrayMoveMutate = (array, from, to) => {
 	const startIndex = to < 0 ? array.length + to : to;
-	const item = array.splice(from, 1)[0];
-	array.splice(startIndex, 0, item);
+
+	if (startIndex >= 0 && startIndex < array.length) {
+		const item = array.splice(from, 1)[0];
+		array.splice(startIndex, 0, item);
+	}
 };
 
-const arrayMove = (array, from, to) => {
-	array = array.slice();
+module.exports = (array, from, to) => {
```

Comment:
IMO it's cleaner to keep the function named.

Reply:
I don't like the added complexity of defining a function just for it to be referenced once.

Reply:
Anonymous functions are bad for debugging. You get only position in stack trace without function name.
Same problem with default exports in general. Named exports help to avoid the problem.

Reply:
From further examination of other Sindre Sorhus modules, I can confirm that using named function, is in fact a good idea.